.bazelrc

```diff
@@ -1,3 +1,6 @@
+# Container Runtime
+build --action_env=CONTAINER_CMD
+build --action_env=XDG_RUNTIME_DIR
```

Comment:
Do we need to set these variables?

Please add them to the README.md.
https://github.com/kubev2v/forklift/blob/main/README.md

Reply:
No, those variables are taken from the environment, if `CONTAINER_CMD` is not set, the container runtime is detected automatically. `XDG_RUNTIME_DIR` is used by podman so it must exist in the environment, if it doesn't it means the runtime is docker and the variable is not needed/used. I will add `CONTAINER_CMD` to the README.